services/movie_session.py

```diff
@@ -24,11 +24,16 @@ def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
     return MovieSession.objects.get(id=movie_session_id)
 
 
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    movie_session = MovieSession.objects.get(id=movie_session_id)
```

Comment:
Why not reuse `get_movie_session_by_id` function defined above to avoid code duplication?